### DISCUSSION TITLE
Virtual Desktop

### DISCUSSION THREAD

**Discussion by macko99** [AzureCommunityPL/group#322] [Pomoc] [answered]
Witam, potrzbuję porady jak najekonomicznej postwić maszynę wirtualną z windowsem (bardzo niskie wymagania, 1 procesor, 2 GB RAM-u) tak aby możliwa była na niej praca zdalna poprzed RDP. Chciałbym aby połaczenie z maszyną było zabezpieczone (np. VPN) oraz możliwa była praca ze środowisk windows/macos. Posiadam licencje Windows 10 E3. Maszyna będzie używana tylko przez jednego user'a. Na jakiego rzędu koszt miesięczy mam liczyć? Z góry dziękuję za odpowiedzi.

**Comment by smereczynski** [comment, marked as answer]
Odpalasz VM z Windows 10 Pro/Enterprise i zaznaczasz sobie, jeśli robisz to przez portal, że posiadasz licencję na wykorzystanie w środowisku multitenant. Jeśli PS lub ARM, to rzuć okiem tutaj: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/windows-desktop-multitenant-hosting-deployment?WT.mc_id=AZ-MVP-5001618

P.S. 1 vCPU i 2GB RAM to trochę mało dla Windowsa z GUI ;)
P.S. 2. VPN to ~€20/mc.
